src/libexpr/primops/fetchClosure.cc

```diff
@@ -208,7 +208,7 @@ static void prim_fetchClosure(EvalState & state, const PosIdx pos, Value * * arg
 }
 
 static RegisterPrimOp primop_fetchClosure({
-    .name = "__fetchClosure",
+    .name = "fetchClosure",
```

Comment:
Is there any particular reason that this is also added to the global scope?

Reply:
sorry, I don't really know what the difference is here, they're all in the global scope anyway, you just have to type `__` before them, no?
I just tried to match it to the other `fetch*` functions that are available without prefix:

```
❮ nix eval --expr 'map __typeOf [fetchGit fetchMercurial fetchTarball fetchTree]'
[ "lambda" "lambda" "lambda" "lambda" ]
```

Reply:
I don't think it's explicated anywhere, but the general convention is to only put `__`-ed things at toplevel to keep the namespace as clean as can be (and even these `__` versions shouldn't be here imho). The other ones are either historical artefacts or anomalies.

Reply:
There is special handling of the `__*` names; I have been meaning to replace this sneaky string mangling with something more declarative, but didn't get around to it yet.

Reply:
Isn't `fetchTree` also fairly new and has no prefix? I don't mind either way, will revert this as well :)